INSTALL

```diff
@@ -7,7 +7,7 @@ Use the autogen script to prepare the build environment.
     make
 
 Precompiled binaries are available at github, see
-https://github.com/dashproject/dash-binaries
+https://github.com/dashpay/dash/releases
```

Comment:
That's not true actually, `.../releases` only have zipped/tared sources. Correct URL for precompiled binaries is https://github.com/dashpay/dash-binaries
This needs to be fixed in 12.1 too.